### ISSUE TITLE
ONNX output mismatch with PyTorch. InstanceNormalization layer not using running_mean and running_var

### ISSUE THREAD

**Issue by ravi9** [pytorch/pytorch#61430]
## 🐛 Bug
ONNX output mismatch with PyTorch. After converting the PyTorch model to ONNX, the outputs mismatch. Please see [compare-pyt-onnx-single-infer.ipynb](https://github.com/ravi9/BrainMaGe/blob/master/compare_onnx/compare-pyt-onnx-single-infer.ipynb)
The running_mean and running variance are not used based on the warning messages printed. How do we make sure the ONNX runtime uses these.
I appreciate any suggestions. Thank you.

## To Reproduce

- Please follow the following steps below to create the conda env. 
- Please see [compare-pyt-onnx-single-infer.ipynb](https://github.com/ravi9/BrainMaGe/blob/master/compare_onnx/compare-pyt-onnx-single-infer.ipynb) to reproduce the error.

```
git clone https://github.com/ravi9/BrainMaGe.git
cd BrainMaGe
git lfs pull
conda env create -f requirements.yml # create a virtual environment named brainmage
conda activate brainmage # activate it
python setup.py install # install dependencies and BrainMaGe

cd compare_onnx
python compare_torch_onnx_il.py
# Or see compare-pyt-onnx-single-infer.ipynb
```

### Expected behavior
Outputs from PyTorch model and ONNX model should match.

## Environment

 - PyTorch Version : 1.5.1
 - OS (e.g., Linux): Ubuntu 18.04
 - How you installed PyTorch (`conda`, `pip`, source): conda
 - Build command you used (if compiling from source): n/a
 - Python version: 3.6.10
 - CUDA/cuDNN version: n/a
 - GPU models and configuration: n/a
 - Any other relevant information:

## Additional context
In [compare_torch_onnx_il.py](https://github.com/ravi9/BrainMaGe/blob/master/compare_onnx/compare_torch_onnx_il.py) , we compared the output after the InstanceNormalization layer where the output starts to differ.
The running_mean and running variance are not used based on the warning messages printed. How do we make sure the ONNX runtime uses these.

```
[W:onnxruntime:, graph.cc:3211 CleanUnusedInitializers] Removing initializer 'ins.in_0.running_mean'. It is not used by any node and should be removed from the model.
[W:onnxruntime:, graph.cc:3211 CleanUnusedInitializers] Removing initializer 'ins.in_0.running_var'. It is not used by any node and should be removed from the model.
```

<img width="857" alt="Screen Shot 2021-07-07 at 5 11 37 PM" src="https://user-images.githubusercontent.com/4837253/124991945-2d02b380-dff7-11eb-9736-5ca1c3f6648c.png">

cc @BowenBao @neginraoof

**Comment by shubhambhokare1** [comment]
Hi @ravi9, can you try using the latest version of pytorch nightly, there have been a few updates when using running_mean and running_var in case of InstanceNorm

**Comment by hwangdeyu** [comment]
Hi @ravi9, `InstanceNormalization` op in [ONNX standard](https://github.com/onnx/onnx/blob/master/docs/Changelog.md#InstanceNormalization-6) is not support `mean` and `var` inputs.

Just as @shubhambhokare1 said, we have [a PR](https://github.com/pytorch/pytorch/pull/60538) to support this situation by combining other ops such as `BatchNormalization'. This PR should fix your issue.

**Comment by ravi9** [comment]
Thank you @shubhambhokare1 and @hwangdeyu. I will test with the latest version of PyTorch nightly.

**Comment by junwenwu** [comment]
@shubhambhokare1 @hwangdeyu We tested from our end and this PR did fix our issue. Thanks!

**Comment by shubhambhokare1** [comment]
PR from @hwangdeyu fixes issue. Closing for now.
